WordPress/Classes/Services/BlogJetpackSettingsService.swift

```diff
@@ -18,10 +18,9 @@ struct BlogJetpackSettingsService {
             return
         }
         guard let remoteAPI = blog.wordPressComRestApi(),
-            let blogDotComId = blog.dotComID as? Int,
-            let blogSettings = blog.settings else {
-                success()
-                return
+            let blogDotComId = blog.dotComID as? Int else {
```

Comment:
I'm surprised this `guard` block co-exists with `supports(.jetpackSettings)` – I'd expect that for a blog to support that it'd need all the other stuff too 🤨

The `success()` here thus feels like a bug – if a block allegedly supports `jetpackSettings`, should we call `success()` if it turns out not to? 🤔